style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: rgb(255, 0, 0);
+  text-align: center;
+}
```

Comment:
- [ ] Please your Pll Request **_title_** and **Summary** are not professionally done. Kindly edit them to reflect this project. Eg. 
- Title: Hello Microverse Project
- Summary: In this pull request, I added the following: list the things you did.